integration-test/utils/datafactory/schemas/team.schema.ts

```diff
@@ -0,0 +1,93 @@
+import { linkSchema } from "./link.schema"
+export const teamSchema = {
+    $ref: '#/definitions/teamSchema',
+    definitions: {
+        teamSchema: {
+            "type": "object",
+            "properties": {
+                "title": {
```

Comment:
I can see from Schema of Team Page API, that the properties from the first level of JSON include `id`, `page`, `contact`, `membersByType` properties instead of `title`, etc.
<img width="389" alt="Screenshot 2025-01-01 at 11 43 47" src="https://github.com/user-attachments/assets/bfef05db-63d5-4cb0-98c6-9b4944cb90e4" />